addons/fleet/security/ir.model.access.csv

```diff
@@ -21,3 +21,5 @@ fleet_service_type_access_right,fleet_service_type_access_right,model_fleet_serv
 access_fleet_vehicle_cost,access_fleet_vehicle_cost,model_fleet_vehicle_cost,fleet_group_manager,1,1,1,1
 access_mail_activity_type_fleet_manager,mail.activity.type.fleet.manager,mail.model_mail_activity_type,fleet.fleet_group_manager,1,1,1,1
 access_fleet_vehicle_assignation_log_fleet_group_user,fleet_vehicle_assignation_log fleet_group_user,fleet.model_fleet_vehicle_assignation_log,fleet.fleet_group_user,1,1,1,1
+access_res_users_fleet_manager,res.users.fleet.manager,base.model_res_users,fleet.fleet_group_manager,0,1,0,0
+access_res_users_fleet_group_user,res.usersfleet_group_user,base.model_res_users,fleet.fleet_group_user,0,1,0,0
```

Comment:
This is a security issue and can't be done! A fleet manager could then add himself to any access group (e.g. Administrator). Or change the login and password of other users.

Reply:
He will not have read permission, he will not be able to see the list of the users to modify it.
This is the only way to solve this error for now.
The error comes from the very backend of odoo its not from the module itself, there is somthing very deep in odoo code that make the create function affect the user of the partner selected as driver.
However i agree with you its a security issue but it will need that the fleet user/manager to be a professional odoo develober to use it :)